Orange/widgets/regression/owsgdregression.py

```diff
@@ -180,6 +162,25 @@ def _on_func_changed(self):
         for spin, enabled in zip(self._func_params, mask):
             spin.setEnabled(enabled)
 
+    def send_report(self):
+        items = OrderedDict()
+        items['Loss function'] = self.LOSS_FUNCTIONS[self.loss_function]
+        if self.LOSS_FUNCTIONS[self.loss_function] != "Squared loss":
+            items['Loss function'] = items['Loss function'] + ",\
+                                     ε={}".format(self.epsilon)
+        items['Penalty'] = self.PENALTIES[self.penalty_type]
+        if self.PENALTIES[self.penalty_type] == "Elastic net (L1 and L2)":
+            items['Penalty'] = items['Penalty'] + (": L1 : L2 = {} : {}".
+                               format(self.l1_ratio, 1.0 - self.l1_ratio))
+        items['Penalty'] = items['Penalty'] + ', α={}'.format(self.alpha)
+        items['Learning rate'] = self.LEARNING_RATES[self.learning_rate] + ",\
+                                 η0={}".format(self.eta0)
+        if self.LEARNING_RATES[self.learning_rate] != "Inverse scaling":
+            items['Learning rate'] = items['Learning rate'] + (", power_t={}".
+                                     format(self.power_t))
+        items['No of iterations'] = self.n_iter
```

Comment:
No need for abbreviation here: Change "No of iterations" to "Number of iterations".